.nvmrc

```diff
@@ -0,0 +1 @@
+16
```

Comment:
use nvmrc, and upgrade to node 16

Reply:
What's the reasoning? And why in a CQL PR?

Reply:
I was updating the openapi yaml, so I took the opportunity to update the build which mostly does validation while I was at it.  I probably should have separated them into different PRs, but didn't.